tests/functional/timeout-test/source/data/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "db": {
+      "host": "${DB_HOST}",
+      "serverName": "${DB_SERVER_NAME}",
+      "port": "${DB_PORT}",
+      "dbLocale": "${DB_LOCALE_VALUE}",
+      "database": "${DB_DATABASE}",
+      "user": "${DB_USER}",
+      "#password": "${DB_PASSWORD}",
+      "odbcTimeout": 1
```

Comment:
můžeš tam dát spíš `queryTimeout`?

Reply:
Stačí takhle v tom configu nebo všude?

Reply:
určitě všude, ať to koresponduje s tím c je v configu

Reply:
@ondrajodas ok, je to všude :)